.github/workflows/development.yml

```
@@ -41,3 +84,38 @@ jobs:
         run: pip install tox
       - name: Run integration tests
         run: tox -e test-integration -- -m smoke
+
+  build:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        python: ["3.9"]
+    steps:
+      - uses: actions/checkout@v4
+      - name: Set up Python
+        uses: actions/setup-python@v5
+        with:
+          python-version: ${{ matrix.python }}
+      - name: Install dependencies
+        run: pip install toml loguru tox
+      - name: Set environment variables
+        run: |
+          echo "GUIDELLM_BUILD_TYPE=dev" >> $GITHUB_ENV
+          echo "GUIDELLM_BUILD_NUMBER=${{ github.event.pull_request.number }}" >> $GITHUB_ENV
+      - name: Build the package
+        run: tox -e build
+      - name: Upload build artifacts
+        id: artifact-upload
+        uses: actions/upload-artifact@v4
+        with:
+          name: build-artifacts
+          path: dist/*
+          compression-level: 6
+          if-no-files-found: error
+          retention-days: 30
+      - name: Comment Install instructions
+        uses: mshick/add-pr-comment@v2
+        with:
+          message: |
+            Build artifacts (.whl and .tar.gz) are available for download for up to 30 days.
+            They are located at https://api.github.com/repos/neuralmagic/guidellm/actions/artifacts/${{ steps.artifact-upload.outputs.artifact-id }}
```

Do you just want the download URL?

```suggestion
            They are located at ${{ steps.artifact-upload.outputs.artifact-url }}
```